osu.Game/Screens/Backgrounds/BackgroundScreenDefault.cs

```diff
@@ -22,23 +26,36 @@ public class BackgroundScreenDefault : BackgroundScreen
 
         private string backgroundName => $@"Menu/menu-background-{currentDisplay % background_count + 1}";
 
-        private Bindable<User> user;
-        private Bindable<Skin> skin;
+        public readonly Bindable<MainMenuBackgroundMode> BackgroundMode = new Bindable<MainMenuBackgroundMode>();
+        public Bindable<User> User = new Bindable<User>();
```

Comment:
private

Reply:
I made them public for test purposes

Reply:
At most we use protected for testing, but looking at your tests I believe they are not required to be public (you should be changing the DI sources instead)